zero/common/gpt2.py

```diff
@@ -229,7 +229,5 @@ def gpt2_builder():
 
     CONFIG['dataset'] = os.environ['DATA']
     CONFIG['tokenizer'] = os.environ['TOKENIZER']
-    if CONFIG['model'].get('numel', None) is None:
-        CONFIG['model']['numel'] = get_model_size(build_model())

```

Comment:
Deletion of 'numel' of `CONFIG` will lead to error as the code after will access this value.